docs/backend-requests/making/jwt-templates.mdx

```diff
@@ -160,7 +160,65 @@ For example, you could use interpolation to build the user's full name:
 }
 ```
 
-Interpolated shortcode substituted values will always be casted to strings. For example, if the user does not have a last name associated, the above full name value would be "null John".
+Interpolated shortcodes will always result to string values. For example, if the user does not have a last name associated, the above full name value would be "null John".
+
+### Coalescing
```

Comment:
❓ I think `coalescing` is more closely related to nullish values: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing while we seem to be taking a slightly broader approach with falsy value categorization. Perhaps going with simply `conditional expression` is less ambiguous as a term (also taking into account possible extensions in the future, e.g. empty strings).

Reply:
Yes, great call out!

Reply:
Changed to "conditional expressions". PTAL